packages/host/src/Site.tsx

```diff
@@ -40,11 +40,12 @@ const Heading: FC = ({ children }) => (
 
 const NotifyOnLogin = () => {
   const { success } = useNotification();
-  const { store } = useAuthContext();
+  const { store, storeId } = useAuthContext();
```

Comment:
offroad: when testing the logged out alert, I noticed that when the `refreshToken` happened, the user received two toasts to say 'welcome to xx store' even though they have already been notified. Changing the reference here removes the duplication and removes the notification-on-refresh-token, which should not have been happening